src/Security/Authentication/Cookies/samples/CookieSessionSample/Startup.cs

```diff
@@ -14,14 +14,12 @@ public class Startup
     {
         public void ConfigureServices(IServiceCollection services)
         {
-            services.AddSingleton<MemoryCacheTicketStore>();
-
             // This can be removed after https://github.com/aspnet/IISIntegration/issues/371
             services.AddAuthentication(options =>
             {
                 options.DefaultAuthenticateScheme = CookieAuthenticationDefaults.AuthenticationScheme;
                 options.DefaultChallengeScheme = CookieAuthenticationDefaults.AuthenticationScheme;
-            }).AddCookie<MemoryCacheTicketStore>((o, ticketStore) => o.SessionStore = ticketStore);
+            }).AddCookie(o => o.SessionStore = new MemoryCacheTicketStore());
```

Comment:
Can you show the workaround here

Reply:
Sure, it just is using the options builder api instead